packages/authentication/package.json

```diff
@@ -3,19 +3,17 @@
   "version": "4.0.0-alpha.30",
   "description": "A LoopBack component for authentication support.",
   "engines": {
-    "node": ">=6"
+    "node": ">=8"
   },
+  "main": "dist/index.js",
```

Comment:
❗️ This is critical ❗️ 

I think this will break monorepo-wide refactorings in VSCode. Before your change, the TypeScript server was seeing `packages/authentication/index.ts` and ignoring any `.js`/`.d.ts` files produced by our build. With your change in place, the TypeScript server will see `packages/authentication/dist/index.js` (and `index.d.ts`), which will hide the original TypeScript sources.

Here are things that will stop working if my assessment is correct:

 - "Go to definition" across packages (e.g. find usage of `@inject` in `authentication` package, press F12) will jump to `.d.ts` file in `dist` instead of the original `.ts` file in `src`.

 - Refactorings like "rename" will not change all places using the renamed entity. Two different scenarios to verify: rename at the place where the entity is defined, rename at the place where the entity is used.

Please verify whether the three scenarios mentioned above work with your change in place. If they do, then my understanding of TypeScript integration in VSCode is no longer correct and you can ignore my comment.

Reply:
Good catch! The first scenario works, but the other two do not. Would ignoring those files in their new locations fix this issue? (will try and check back)